ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/WSQueryEndpoint.java

```diff
@@ -279,16 +272,28 @@ private KsqlRequest parseRequest(final MultiMap requestParams) {
     }
   }
 
+  private void attachCloseHandler(final ServerWebSocket websocket,
+                                  final WebSocketSubscriber<?> subscriber) {
+    websocket.closeHandler(v -> {
+      subscriber.close();
+      final String msg = String.format("Websocket %s closed, reason: %s,  code: %s)",
+              websocket.textHandlerID(),
+              websocket.closeReason(),
+              websocket.closeStatusCode());
+      log.debug(msg);
```

Comment:
I recommend using the `log.debug` parameters to pass variables to the string. Like:
```
log.debug("Websocket {} closed, reason: {},  code: {})", 
        websocket.textHandlerID(), 
        websocket.closeReason(), 
        websocket.closeStatusCode());
```

Passing the variables to the LOG instead of using the String.format() has the benefit that if DEBUG is disabled (in most cases), then the string will not be formatted. However, if using String.format(), then the string is always formatted even if the log doesn't print it because debug is disabled.

Reply:
Ah yes, was looking for something like this!